debug/debug.go

```diff
@@ -14,101 +14,109 @@ import (
 )
 
 var opts struct {
+	logger *log.Logger
 	tags   map[string]bool
 	breaks map[string]bool
 	m      sync.Mutex
 }
 
-var debugLogger = initDebugLogger()
```

Comment:
There is a reason that the `debugLogger` variable is not in the `opts` struct
and that it is initialized this way: In Go, variables are initialized first,
then the `init()` functions are called.

Initializing `debugLogger` this way makes sure that it is already available in
`init()` functions of other packages.

Relevant part of the spec is at https://golang.org/ref/spec#Package_initialization